modules/Hoot/ui/sidebar/layerConflate.js

```diff
@@ -106,7 +111,8 @@ class LayerConflate extends SidebarForm {
             .selectAll( '.thumb' )
             .data( layers ).enter()
             .append( 'div' )
-            .attr( 'class', d => `thumb round _icon data light contain inline fill-${ d.color }` );
+            .attr( 'class', d => `thumb round _icon data light contain inline fill-${ d.color }` )
+            .attr( 'id', d => `${d.refType}` );
```

Comment:
the sidebar layer form already uses this `id`.  might be better to put it in the `class`.
```
.attr( 'class', d => `${d.refType} thumb round _icon data light contain inline fill-${ d.color }` );
```